docs/content/preview/drivers-orms/smart-drivers.md

```diff
@@ -188,25 +188,30 @@ If no servers are available, the request may return with a failure.
 Consider a cluster with three primary nodes across three zones (zoneA, zoneB, and zoneC); and three read replica nodes in zoneB, zoneC, and zoneD.
 
 The following shows how nodes are selected for new connections, depending on the load balance and topology key settings.
+The property names used are for the Java smart driver; the property names for other smart drivers may be slightly different.
+The property `fallback-to-topology-keys-only` is ignored when either the `topology-keys` empty or when `load-balance` is set to `prefer-primary` or `prefer-rr`.
 
 {{<tabpane text=true >}}
```

Comment:
The new tab setup doesn't seem like an improvement - there are 3 options, was the original found to be confusing?

Reply:
@ashetkar

Reply:
That's still incomplete @ddhodge, I have some changes not pushed yet. Please allow me another day to\ push those (seeing some formatting issues) and update this thread.

Reply:
@ashetkar Is it complete now?

Reply:
Yes. I've pushed another commit today.